jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+work normally, though they will need to be installed and called using custom commands.
+
+## Code signing
+[fastlane](https://fastlane.tools) and [Shenzhen](http://nomad-cli.com/#shenzhen)
+are pre-installed on the container image. It is recommended to use fastlane to
+build, sign and distribute your iOS app to beta-testers.
+
+The fastest way to get code signing working on CircleCI is to follow
+these steps:
+
+1. Upload your provisioning profile (`.mobileprovision`) and private key (`.p12`)
+   files in **Project Settings > iOS Code Signing**.
+1. Set `GYM_CODE_SIGNING_IDENTITY` environment variable
+   to match your code-signing identity, ie `"iPhone Distribution: Acme Inc."`.
+1. Build with `gym` and deploy with `ipa`.
+
+## Customizing Your Build
+You have complete flexibility to customize what runs in your build. CircleCI runs tests from the command line with the [`xcodebuild`](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/xcodebuild.1.html)
+command by default. This is a tool developed by Apple that is a 
+stable and functional option for building and testing your macOS project.
+
+The following command is representative of how CircleCI builds an iOS project:
+
+```
+    steps:
+      - run:
+          name: homebrew dependencies
+          command: |
+            set -o pipefail &&
+            xcodebuild
+              CODE_SIGNING_REQUIRED=NO
+              CODE_SIGN_IDENTITY=
+              PROVISIONING_PROFILE=
+              -sdk iphonesimulator
+              -destination 'platform=iOS Simulator,OS=9.0,name=iPhone 6'
+              -workspace MyWorkspace.xcworkspace
+              -scheme "My Scheme"
+              clean build test |
+            tee $CIRCLE_ARTIFACTS/xcode_raw.log |
+            xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
+```
+
+The destination can be selected from the simulators the are pre-installed in CircleCI
+build images.
+
+In some situations you might also want to build with [`xctool`](https://github.com/facebook/xctool),
+an alternative build tool. Please mind that some of the `xcodebuild` functionality might not be
+supported by `xctool`. Here is an example of an `xctool` build command:
+
+```
+    steps:
+      - run:
+          name: homebrew dependencies
+          command: |
+            - xctool
+            -reporter pretty
+            -reporter junit:$CIRCLE_TEST_REPORTS/xcode/results.xml
+            -reporter plain:$CIRCLE_ARTIFACTS/xctool.log
+            CODE_SIGNING_REQUIRED=NO
+            CODE_SIGN_IDENTITY=
+            PROVISIONING_PROFILE=
+            -destination 'platform=iOS Simulator,name=iPhone 6,OS=latest'
+            -sdk iphonesimulator
+            -workspace MyWorkspace.xcworkspace
+            -scheme "My Scheme"
+            build build-tests run-tests
+```
+
+### Setting Environment Variables
```

Comment:
This section describes functionality provided via inference, there is no inference in 2.0, so we should remove this section completely